domains/certificates/Shared/DataContext/Migrations/ApplicationDbContextModelSnapshot.cs

```diff
@@ -193,8 +193,9 @@ protected override void BuildModel(ModelBuilder modelBuilder)
                     b.Property<int>("ActorType")
                         .HasColumnType("integer");
 
-                    b.Property<Guid>("EntityId")
-                        .HasColumnType("uuid");
+                    b.Property<string>("EntityId")
+                        .IsRequired()

```

Comment:
Why is it marked as required? It wasn't before?

Reply:
Because it's not a nullable field

Reply:
ok